cpp/src/parquet/reader_test.cc

```diff
@@ -58,6 +58,63 @@ std::string nation_dict_truncated_data_page() {
   return data_file("nation.dict-malformed.parquet");
 }
 
+std::string hadoop_lz4_compressed() { return data_file("hadoop_lz4_compressed.parquet"); }
+
+// TODO: Assert on definition and repetition levels
+template <typename DType, typename ValueType>
+void ASSERT_BATCH_EQUAL(std::shared_ptr<TypedColumnReader<DType>> col, int64_t batch_size,
```

Comment:
Use CamelCase on function names. Also, not sure "Batch" is the right word (there may be confusion with Arrow record batches).

Reply:
"Batch" is in the name of the `ColumnReader` method `ReadBatch`

Reply:
I see. The problem is that functions named "AssertBatchEqual" (or "AssertBatchesEqual") already exist on the Arrow side, which promises to be confusing. How about "AssertColumnValues" or something?

Reply:
Good point - changed to AssertColumnValues